src/main/java/AutomationPracticeWithToolsQASite/Pages/AutomationFormPage.java

```diff
@@ -30,8 +28,15 @@ public void enterLastName(String lastName) {
         final WebElement firstNameTab = this.waitMethods.waitForElementToBeVisible(this.lastName);
         firstNameTab.sendKeys(lastName);
     }
-    public String validateLastName() {
+    public String getLastName() {
         final WebElement enteredFirstName = this.waitMethods.waitForElementToBeVisible(this.lastName);
         return enteredFirstName.getAttribute("value");
     }
+
+    public void clickSimpleButton() {
+        waitMethods.sleep(4);
```

Comment:
Make the heavens fall.
Don't use sleep here. 
Construct a waiter if you want.
for loop with 0.5 sec waiters - ignore exception

Reply:
Will do!